src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -0,0 +1,92 @@
+"""
+    NormInfinityBridge{T}
+
+The `NormInfinityCone` is representable with LP constraints, since ``t \\ge \\max_i |x_i|`` if and only if ``t \\ge x_i`` and ``t \\ge -x_i`` for all ``i``.
+"""
+struct NormInfinityBridge{T} <: AbstractBridge
+    ge_index::CI{MOI.VectorAffineFunction{T}, MOI.Nonnegatives}
+end
+function bridge_constraint(::Type{NormInfinityBridge{T}}, model, f::MOI.VectorOfVariables, s::MOI.NormInfinityCone) where T
+    return bridge_constraint(NormInfinityBridge{T}, model, MOI.VectorAffineFunction{T}(f), s)
+end
+function bridge_constraint(::Type{NormInfinityBridge{T}}, model, f::MOI.VectorAffineFunction{T}, s::MOI.NormInfinityCone) where T
+    f_scalars = MOIU.eachscalar(f)
+    t = f_scalars[1]
+    d = MOI.dimension(s)
+    new_f = MOIU.vectorize(vcat([MOIU.operate(-, T, t, f_scalars[i]) for i in 2:d], [MOIU.operate(+, T, t, f_scalars[i]) for i in 2:d]))
```

Comment:
You can do `ub = MOIU.operate!(-, T, f_scalars[2:d])` and then `MOIU.operate_output_index!(+, i, T, ub, t)`